### ISSUE TITLE
Remove finding id/Y/N uniqueness validation

### ISSUE THREAD

**Issue by jadudm** [GSA-TTS/FAC#4385]
# TL;DR

The uniqueness validation over [finding id | is_*] fields works for *many* audits, but not for *some*. We estimate the "some" to be ~500/year, based on historical data. 

We do not need to remove the validation from the codebase (yet), as we may revise it. However, we should remove it from the list of validations that are run at submission time.

- [ ] Remove the finding/is_* validation from the list of validations run at submission time (and anywhere else)
- [ ] Make sure tests pass locally; remove/comment out tests that exercised this validation.
- [ ] Confirm in staging

# Explanation

Consider

```
| AWARD-001 | 2023-001 | Y | N | N |
| AWARD-002 | 2023-002 | Y | Y | N |
| AWARD-003 | 2023-001 | Y | Y | N |
```

In this scenario, finding 2023-001 is both `YNN` and `YYN`. We have been considering the Y/N values as "apples-to-apples." That is, we thought all the answers were of the same character.

It appears, however, that some of these questions would apply to all monies from a given award, while some questions would apply to a particular expenditure line only. For example, if an entity has bad vendor management, that may apply across many awards, but it might be for one award only that they have questioned costs. This would lead to a situation where the Y/N values were consistent across multiple awards with the same finding number, but for *one* of them, there would be a difference (questioned_costs).

This is most likely to come up in large audits. Hence, the validation passes *most* audits, but when it doesn't, it is likely in contradiction with auditing practice. (As we do more research, relevant UG references will be attached.)

In short, it appears that *some* of the Y/N findings values are of the same kind ("apples"), and some are not ("oranges"). We need to determine which-is-which before bringing this validation back.

We 

# Next steps

We may need to revisit this validation with both auditors and Federal colleagues to better understand what this information means in the SF-SAC before putting a revised validation in place.

**Comment by jadudm** [comment]
Likely relevant UG:

* https://www.ecfr.gov/current/title-2/part-200/section-200.516#p-200.516(b)(6)
* https://www.ecfr.gov/current/title-2/subtitle-A/chapter-II/part-200#p-200.510(b)
* https://www.ecfr.gov/current/title-2/part-200/subject-group-ECFRea73e47c9a286e6#p-200.515(d)(3)(i)
